.github/actions/integration-test/action.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Run Integration Test"
+description: "Runs integration tests in a kinD cluster on a github action"
+
+inputs:
+  aws_role:
+    description: "role to acquire from aws"
+    required: true
+  vpc_id:
+    description: "aws vpc id to use for the test"
+    required: true
+  account_id:
```

Comment:
How are the inputs passed in? Will the account_id and role still be marked as secret?

Reply:
Yes - all inputs are elided in the same way (I think secrets are hidden based off of just pattern matching, so the origin doesn't really matter). You can see this in the [raw output](https://pipelines.actions.githubusercontent.com/serviceHosts/a48c4af4-3153-4cba-83b6-d8bb05fa32db/_apis/pipelines/1/runs/642/signedlogcontent/2?urlExpires=2022-12-22T23%3A45%3A52.5792756Z&urlSigningMethod=HMACV1&urlSignature=p2AbNwBv1CYj2VuKlFHxue9NdorPI7Sftx8CEkbh7fA%3D).